AudioQnA/kubernetes/manifests/xeon/audioqna.yaml

```diff
@@ -101,7 +101,7 @@ spec:
       - envFrom:
         - configMapRef:
             name: audio-qna-config
-        image: opea/whisper:latest
+        image: ${REGISTRY:-opea}/whisper:${TAG:-latest}
```

Comment:
I'm afraid the changes to manifest doesn't work, including the benchmark manifest files.

Reply:
I will check. Thanks!